go.mod

```diff
@@ -15,10 +15,10 @@ replace (
 )
```

Comment:
I think CI fails due to compatibility with upstream packages cause of the replaces here.

Could you map these k8s packages to `v0.32.2` instead of `v0.31.1` and see if it helps?